impeller/renderer/backend/metal/render_pass_mtl.h

```diff
@@ -13,6 +13,64 @@
 
 namespace impeller {
 
+//-----------------------------------------------------------------------------
+/// @brief      Ensures that bindings on the pass are not redundantly set or
+///             updated. Avoids making the driver do additional checks and makes
+///             the frame insights during profiling and instrumentation not
+///             complain about the same.
+///
+///             There should be no change to rendering if this caching was
+///             absent.
+///
+struct PassBindingsCache {
```

Comment:
Move to its own TU and rename as `PassBindingsCacheMTL`.

Reply:
Done